website/integrations/services/sharepoint-se/index.md

```diff
@@ -0,0 +1,377 @@
+| authentik Application Slug                         | `auth.applicationSlug`               | sharepoint-se                                                                          |
+| authentik OIDC Name                                | `auth.providerName`                  | OIDC-SP                                                                                |
+| authentik OIDC Configuration URL                   | `auth.providerConfigURL`             | https://authentik.company/application/o/sharepoint-se/.well-known/openid-configuration |
+| authentik OIDC Client ID                           | `auth.providerClientID`              | 0ab1c234d567ef8a90123bc4567890e12fa3b45c                                               |
+| authentik OIDC Redirect URIs                       | `auth.providerRedirectURI`           | https://sharepoint.company/.\*                                                         |
+| (Optional) authentik LDAP Outpost URI              | `ldap.outpostURI`                    | ak-outpost-ldap.authentik.svc.cluster.local                                            |
+| (Optional) authentik LDAP Service Account          | `ldap.outpostServiceAccount`         | cn=ldapservice,ou=users,dc=ldap,dc=goauthentik,dc=io                                   |
+| (Optional) authentik LDAP Service Account Password | `ldap.outpostServiceAccountPassword` | mystrongpassword                                                                       |
+| SharePoint Default Web Application URL             | `sp.webAppURL`                       | https://sharepoint.company                                                             |
+| SharePoint Trusted Token Issuer Name               | `sp.issuerName`                      | Authentik                                                                              |
+| SharePoint Trusted Token Issuer Description        | `sp.issuerDesc`                      | authentik IDP                                                                          |
+
+## authentik configuration
+
+### Step 1: Create authentik Open ID Property Mappings
+
+SharePoint requires additional properties within the openid and profile scopes in order to operate OIDC properly and be able to map incoming authentik OID Claims with Microsoft Claims.
+
+Additional information from Microsoft documentation:
+
+-   https://learn.microsoft.com/en-us/entra/identity-platform/id-tokens#validate-tokens
+-   https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#payload-claims
+
+#### Add an openid scope mapping for SharePoint
+
+From the authentik Admin Dashboard:
+
+1. Open **Customisation > Property Mappings** page from the sidebar.
+2. Click **Create** from the property mapping list command bar.
+3. Within the new property mapping form, select **Scope Mapping**.
+4. Click **Next** and enter the following values:
+    - **Name**: SPopenid
+    - **Scope name**: openid
```

Comment:
Is this an option in the dropdown? Or should it be OpenID? :-)

Reply:
In that case it’s the value used by the client it MUST be in lowercase.

![image](https://github.com/goauthentik/authentik/assets/24614045/817a7acc-2105-4505-8abd-7110d6919a65)

Reply:
It’s not a dropdown list, a simple text field.